swift/Samples/Tutorial/Frameworks/Tutorial1Complete/Tutorial1.podspec

```diff
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
   s.source       = { git: 'Not Published', tag: "podify/#{s.version}" }
 
   # 1.7 is needed for `swift_versions` support
-  s.cocoapods_version = '>= 1.7.0.beta.1'
+  s.cocoapods_version = '>= 1.7.0'
```

Comment:
Should this be `1.9.1`?

Reply:
Adding some more info relevant to this upgrade. I'm quoting @amorde here from the modal sample [PR](https://github.com/square/workflow/pull/1090)  

> It's worth keeping something in s.cocoapods_version in case someone tries to pull in workflow while using an old CocoaPods installation, but it definitely doesn't need to be a beta. 1.7.0 should work

Reply:
Thanks for adding the relevant comment @asthatrivedi! If we set it to that version, we'd force developers to update their version of cocoapods to use our library. We should try and use the minimum required version where possible.

Reply:
Makes sense, not sure what's happening with the builds.

Reply:
Workflow should support any CocoaPods above `1.7.0` so this LGTM